VidLoader/VidLoaderTests/Mocks/MockAVAssetDownloadTask.swift

```diff
@@ -9,8 +9,10 @@
 import AVFoundation
 
 final class MockAVAssetDownloadTask: AVAssetDownloadTask {
-
-    init(noUse: Bool? = nil) {}
+    
+    convenience init(noUse: Bool? = nil) {
+        self.init()
```

Comment:
this line make tests run recursively, so they never stop, but was needed to make it compilable